README.md

```diff
@@ -50,9 +99,9 @@ Maximum connection rate per 10 seconds per IP: 40
 Maximum amount of data per 30 second: 100 MB
```

Comment:
maybe it's better to just explain different clusters and refer to https://docs.solana.com/cluster/rpc-endpoints. mention that pre-mainnet lives on devnet as well.